### PR TITLE
Use plural attributes in Transitions and Trajectories

### PR DESCRIPTION
**Pull request by shwang**
Throughout the codebase, renamed `act`=>`acts`, `done`=>`dones`, `rew`=>`rews` when appropriate. (`*obs` remains the same.) This matches the naming convention for vectorized environments' returns in the Baselines codebase.

* [x] Upload new rollout pkls

The tests data rollout files are pretty big now (1 MB each). They roughly the same number of Trajectories as before (~50). I believe that they are bigger because of the new `infos` field, with its 50 empty dictionaries per Trajectory.

### CODE REVIEW

**Comment by shwang** [comment]
Reduced difficulty of test_bc because BC performance seemed to vary quite a bit every time I regenerated CartPole rollouts.

The current rollouts have `return_mean=386` but other times I something like 150 or 200.  
```
(Pdb) p novice_stats
{'n_traj': 10, 'return_min': 10.0, 'return_mean': 20.1, 'return_std': 11.945291959596467, 'return_max': 53.0, 'len_min': 10, 'len_mean': 20.1, 'len_std': 11.945291959596467, 'len_max': 53, 'monitor_return_min': 10.0, 'monitor_return_mean': 20.1, 'monitor_return_std': 11.945291959596467, 'monitor_return_max': 53.0}
(Pdb) p good_stats
{'n_traj': 25, 'return_min': 150.0, 'return_mean': 386.88, 'return_std': 108.72067696625146, 'return_max': 500.0, 'len_min': 150, 'len_mean': 386.88, 'len_std': 108.72067696625146, 'len_max': 500, 'monitor_return_min': 150.0, 'monitor_return_mean': 386.88, 'monitor_return_std': 108.72067696625146, 'monitor_return_max': 500.0}
```

Under the rollouts on master we consistently get `return_mean=500.0` though. In the worst case this change is due to a regression in this PR or #97. I'll take a careful look before we merge this.

File: src/imitation/scripts/config/expert_demos.py [hunk 1]
```diff
@@ -37,6 +37,15 @@ def expert_demos_defaults():
   log_root = os.path.join("output", "expert_demos")  # output directory
 
 
+@expert_demos_ex.config
+def default_end_cond(rollout_save_n_timesteps, rollout_save_n_episodes):
```
Comment (AdamGleave) [inline src/imitation/scripts/config/expert_demos.py]: :+1: nice idea.

**Comment by shwang** [comment]
Previously they got mean 500 returns (master) I think reported the current
mean return in a previous comment on the PR.

**Comment by shwang** [comment]
Squashing into #97 now. Thanks for the review

**Comment by shwang** [comment]
Regression was due to VecNormalize changing the observations in `rollout.save`. #102 provides a fix.

**Pull request merged**